Apps/WebClient/test/functional/e2e/cypress/support/commands.js

```diff
@@ -35,4 +35,14 @@ Cypress.Commands.add("login", (username, password, authMethod = AuthMethod.BCSC)
         cy.get('#kc-login').click()
     }
     cy.get('#subject').should('have.text', 'Health Care Timeline')
-})
\ No newline at end of file
+});
+
+Cypress.Commands.add("checkTimelineHasLoaded", () => 
+{
+    cy.get('[data-testid=timelineLoading]').should('not.exist');
+});
+
+Cypress.Commands.add("closeCovidModal", () => 
+{
+    cy.get('#covid-modal___BV_modal_header_ > .close').click()
```

Comment:
We should be finding the button within the covid modal instead of looking for the generated ID